project.clj

```diff
@@ -34,7 +34,8 @@
   :profiles
   {:dev {:dependencies
          [[figwheel-sidecar   "0.5.0"]
-          [binaryage/devtools "0.9.9"]]}
+          [binaryage/devtools "0.9.9"]]
+         :source-paths ["src" "etc/repl"]}
```

Comment:
Can you add a line above `:source-paths` and break the vector down (just for consistent formatting)